src/main/java/cms/gov/madie/measure/dto/MeasureSearchCriteria.java

```diff
@@ -0,0 +1,19 @@
+package cms.gov.madie.measure.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@Data
+@Builder(toBuilder = true)
+@AllArgsConstructor
+@NoArgsConstructor
+public class MeasureSearchCriteria {
+  private String query;
+  private String model;
+  private Boolean draft;
+  private List<String> excludeMeasures;
```

Comment:
Recommend rename to `excludeByMeasureIds` to clarify the String is expected to be measure IDs.